package.json

```diff
@@ -42,17 +42,17 @@
     "shx": "^0.3.2",
     "ts-jest": "^24.0.2",
     "tslint": "^5.17.0",
-    "typescript": "^3.7.0"
+    "typescript": "^4.1.2"
   },
   "dependencies": {
-    "documentdb": "^1.15.3",
+    "@azure/cosmos": "^3.7.2",
     "durable-functions": "^1.2.4",
```

Comment:
this must be updated as well

Reply:
Should we update to `3.9.3` ?

Reply:
i mean durable functions package

Reply:
Done :) [4d08bd3](https://github.com/pagopa/io-functions-template/pull/8/commits/4d08bd3f10aa0455648959c6c1a15828ec0e0592)